### PR TITLE
Add tooltip on hover for images

### PR DESCRIPTION
**Pull request by LorenDB**
I can't build this myself since I don't have an unlimited Internet connection (and therefore I can't afford to waste it on building [but I *never* make mistakes :wink:]), so if somebody wants to test this out, that would be great.

### CODE REVIEW

**Comment by LorenDB** [comment]
P.S. This is linked to PR #297.

File: resources/qml/delegates/ImageMessage.qml [hunk 1]
```diff
@@ -34,9 +34,12 @@ Item {
 		fillMode: Image.PreserveAspectFit
 
 		MouseArea {
+            hoverEnabled: true
 			enabled: model.data.type == MtxEvent.ImageMessage && img.status == Image.Ready
 			anchors.fill: parent
 			onClicked: timelineManager.openImageOverlay(model.data.url, model.data.id)
+            ToolTip.visible: hovered
+            ToolTip.text: model.data.id
```
Comment (deepbluev7) [inline resources/qml/delegates/ImageMessage.qml:42]: Why do you want to show the event id when hovering over the image?

Comment (LorenDB) [reply resources/qml/delegates/ImageMessage.qml:42]: Is that what that does? Notice that I don't really do QML (just picked up a little for this).

Comment (LorenDB) [reply resources/qml/delegates/ImageMessage.qml:42]: I'll work on it more :)

Comment (deepbluev7) [reply resources/qml/delegates/ImageMessage.qml:42]: You were accessing this member: https://github.com/Nheko-Reborn/nheko/blob/master/src/timeline/TimelineModel.cpp#L355

You probably want to show something different. I would also appreciate it, if you tried to do something similar to Lurkkis design, to be somewhat consistent (i.e. not using a tooltip, but a somewhat transparent overlay with image description, filename, maybe size and a maximize button or so).

You probably need to build that to properly test it.

Comment (LorenDB) [reply resources/qml/delegates/ImageMessage.qml:42]: OK, I'll see what I can do.

**Comment by LorenDB** [comment]
Closed since @kamathmanu is working on this thing.

**Pull request closed without merging**